drivers/net/bnxt/rte_pmd_bnxt.c

```diff
@@ -482,7 +478,7 @@ int rte_pmd_bnxt_mac_addr_add(uint8_t port, struct ether_addr *addr,
 	/* If the VF currently uses a random MAC, update default to this one */
 	if (bp->pf.vf_info[vf_id].random_mac) {
 		if (rte_pmd_bnxt_get_vf_rx_status(port, vf_id) <= 0) {
-			bnxt_hwrm_func_vf_mac(bp, vf_id, (uint8_t *)addr);
+			rc = bnxt_hwrm_func_vf_mac(bp, vf_id, (uint8_t *)addr);
```

Comment:
If this fails, shouldn't we bailout?
Also curious.. do we need to set the filter for the MAC which is also being set as default_MAC ?